### ISSUE TITLE
penalize conf and loc 

### ISSUE THREAD

**Issue by MicBA** [rykov8/ssd_keras#80]
Hi everyone 
in the process of the SSD train, we compute the loss on the location (L1) and the conf (softmax ) 
as i see it in the code they are not normalize 
correct me if i;m wrong but that cannot cause trouble ? 
if one of the loss is greater then the other it will shift the gradient  
is someone try to penalize one of them? which value is good reference ?

other implementation i thought is to keep the loc and conf train in two separate streams 
but this is not the paper implementation 
Thanks !
